recipe/build.sh

```diff
@@ -4,6 +4,7 @@ mkdir build
 cd build
 
 cmake \
+    -DCMAKE_CXX_FLAGS="-DBOOST_ERROR_CODE_HEADER_ONLY" \
```

Comment:
Can you do `export CXXFLAGS="$CXXFLAGS -DDBOOST_ERROR_CODE_HEADER_ONLY"` instead of this?

Reply:
Ok.